app/helpers/companies_helper.rb

```diff
@@ -0,0 +1,2 @@
+module CompaniesHelper
```

Comment:
incorrect module location
is it really needed?

Reply:
true, as of the moment there is no need in that helper, so i removed it